src/components/dev-hub/card-list.js

```diff
@@ -41,71 +41,76 @@ const getThumbnailUrl = media => {
         : media.thumbnailUrl;
 };
 
-// TODO: Update this component to take one array of items and map to cards based
-// on the item type
+const sortCardsByDate = contentList =>
+    contentList.sort((a, b) => {
+        if (new Date(a.publishDate) > new Date(b.publishDate)) return -1;
+        if (new Date(a.publishDate) < new Date(b.publishDate)) return 1;
+        return 0;
+    });
+
+const showArticles = article => (
+    <ArticleCard
+        to={article['slug']}
+        key={article['_id']}
+        image={withPrefix(article['atf-image'])}
+        tags={getTagLinksFromMeta(article)}
+        title={getNestedText(article['title'])}
+        description={getNestedText(article['meta-description'])}
+    />
+);
+
+const showVideos = video => (
+    <VideoModal
```

Comment:
This can just be a `Card` with the `video` and `videoThumbnail` props now

Reply:
For some reason, changing the `image` prop to `videoThumbnail` doesn't render the thumbnail image. Weird because it seems to be working for the `index.js` page